app/controllers/concerns/idv/document_capture_concern.rb

```diff
@@ -2,6 +2,7 @@ module Idv
   module DocumentCaptureConcern
     def override_document_capture_step_csp
       return if params[:step] != 'document_capture'
+      return if IdentityConfig.store.suppress_content_security_policy
```

Comment:
From @zachmargolis: this methods opens up the policy, does it still cause problems for the accessibiltiy tool on this step if we leave it in?

Reply:
@zachmargolis 

ANDI (anyhow) cannot be loaded/used without line 5, despite the open policy, although I am at a lost as to know why at the moment, because the failure is for unsafe-eval which the policy seems to allow. The other errors are curious also, quite possibly because of the policy.xxx_src values which I haven't looked at yet.

![Screen Shot 2022-08-10 at 3 28 11 PM](https://user-images.githubusercontent.com/346917/184003743-77eacec3-639e-4108-86c6-689ba9cd2312.png)